ocaml/xapi/xapi_fuse.ml

```diff
@@ -72,7 +72,8 @@ let light_fuse_and_dont_restart ?(fuse_length = !Xapi_globs.fuse_time) () =
                log_and_ignore_exn Xapi_stats.stop;
                log_and_ignore_exn Rrdd.backup_rrds;
                Thread.delay fuse_length;
-               Db_cache_impl.flush_and_exit (Db_connections.preferred_write_db ()) 0) ());
+               let dbconn = Xapi_stdext_monadic.Opt.unbox (Db_connections.preferred_write_db) in
+               Db_cache_impl.flush_and_exit dbconn 0) ());
```

Comment:
So we deliberately use `Opt.unbox` in this file to fail with an error, instead of `Opt.iter`, like in the other files?

Reply:
I figured since the whole thing is inside a try and `Db_connections.preferred_write_db ()` is already raising on `List.Hd` if there isn't one, this keeps the behaviour the same.